homeassistant/components/coronavirus/sensor.py

```diff
@@ -27,17 +27,21 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
 class CoronavirusSensor(CoordinatorEntity, SensorEntity):
     """Sensor representing corona virus data."""
 
-    name = None
-    unique_id = None
+    _attr_unit_of_measurement = "people"
+    _attr_extra_state_attributes = {ATTR_ATTRIBUTION: ATTRIBUTION}
```

Comment:
It's a bit dangerous with mutable class attributes. If it's modified all instances will inherit the change.

Reply:
Thanks! Adjusted.